setup.py

```diff
@@ -48,7 +49,7 @@
     packages=find_packages(include=["unyt"]),
     test_suite="tests",
     tests_require=test_requirements,
-    python_requires=">=3.5",
+    python_requires=">=3.6",
```

Comment:
Cargo culting a bit but I've been seeing a lot of packages requiring 3.6.1, instead of 3.6.0
I've been less and less motivated to figure out why though, since Python 3.6 is soon to be EOL

Reply:
Easy enough to do and I don't think anyone is going to complain on that. :)